orc8r/cloud/docker/docker-compose.yml

```diff
@@ -44,6 +44,9 @@ services:
 
   postgres:
     image: postgres
+    # Default is 64mb
```

Comment:
In prod we use a managed db, so it's irrelevant. For dev, the db was crashing when I added the 15k subscribers because it was out of memory. This fixes by increasing the shared memory, allowing db to hold all subs